### PR TITLE
Design 2 fixes

### PR DESCRIPTION
**Pull request by letier3110**

### CODE REVIEW

File: src/components/asset-equity-text/asset-equity-text.styles.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { createUseStyles } from '../../styles/create-use-styles';
+
+export const useAssetEquityTextStyles = createUseStyles(() => ({
+  numberText: {
+    color: '#707070'
```
Comment (LendiHop) [inline src/components/asset-equity-text/asset-equity-text.styles.ts:5]: use greyLight const from styles.ts

File: src/components/token-screen-content-container/token-screen-content-container.tsx [hunk 2]
```diff
@@ -16,14 +21,24 @@ const historyComponentIndex = 0;
 
 export const TokenScreenContentContainer: FC<Props> = ({ historyComponent, infoComponent }) => {
   const styles = useTokenScreenContentContainerStyles();
+  const { navigate } = useNavigation();
+  const [, isBakerSelected] = useSelectedBakerSelector();
```
Comment (LendiHop) [inline src/components/token-screen-content-container/token-screen-content-container.tsx:25]: remove this comma

Comment (letier3110) [reply src/components/token-screen-content-container/token-screen-content-container.tsx:25]: i need second element of array of that selector, so its 100% legal operation

**Review by LendiHop — APPROVED** [review]

File: src/components/formatted-amount.tsx [hunk 3]
```diff
@@ -32,14 +32,16 @@ export const FormattedAmount: FC<Props> = ({
...
   return (
     <Text style={style}>
-      {isDollarValue && '≈ '}
+      {!isZeroAmount && isDollarValue && '≈ '}
       {showMinusSign && '- '}
       {showPlusSign && '+ '}
-      {formattedAmount}
-      {isDollarValue ? ' $' : ` ${symbol}`}
+      {visibleAmount}
+      {isDollarValue ? ' $' : isZeroAmount ? '' : ` ${symbol}`}
```
Comment (unixvb) [inline src/components/formatted-amount.tsx:44]: combine into single condition

File: src/components/header/header-token-info/header-token-info.tsx [hunk 4]
```diff
@@ -15,12 +15,16 @@ export const HeaderTokenInfo: FC<Props> = ({ token }) => {
   const styles = useHeaderTokenInfoStyles();
 
   const { name, symbol } = token;
-  const title = isString(name) ? name : symbol;
+  const title = symbol;
```
Comment (unixvb) [inline src/components/header/header-token-info/header-token-info.tsx:18]: remove this const

File: src/components/token-screen-content-container/token-screen-content-container.tsx [hunk 5]
```diff
@@ -16,14 +21,24 @@ const historyComponentIndex = 0;
...
 
   const [segmentedControlIndex, setSegmentedControlIndex] = useState(0);
   const showHistoryComponent = segmentedControlIndex === historyComponentIndex;
 
   return (
     <>
       <View style={styles.headerContainer}>
-        <Text style={styles.headerText}>{showHistoryComponent ? 'History' : 'Info'}</Text>
+        <TouchableOpacity style={styles.delegateContainer} onPress={() => navigate(ScreensEnum.Delegation)}>
+          {!isBakerSelected ? (
```
Comment (unixvb) [inline src/components/token-screen-content-container/token-screen-content-container.tsx:34]: remove !

File: src/components/formatted-amount.tsx [hunk 6]
```diff
@@ -25,20 +25,24 @@ export const FormattedAmount: FC<Props> = ({
...
+  if (isZeroAmount) {
+    return <Text style={style}>- - - -</Text>;
+  }
+
+  const dollarAmount = amount.isPositive()
     ? bigIntClamp(amount, MIN_POSITIVE_AMOUNT_VALUE, new BigNumber(Infinity))
     : bigIntClamp(amount, new BigNumber(-Infinity), MAX_NEGATIVE_AMOUNT_VALUE).abs();
 
   const formattedAmount = isDollarValue ? formatAssetAmount(dollarAmount, 2) : formatAssetAmount(amount);
+  const visibleAmount = formattedAmount;
```
Comment (unixvb) [inline src/components/formatted-amount.tsx:38]: no need in this variable any more

**Review by unixvb — APPROVED** [review]

**Pull request merged**